### ISSUE TITLE
The training results are not good

### ISSUE THREAD

**Issue by Mizar29** [Coordi777/Accelerating-Diffusion-for-SAR-to-Optical-Image-Translation#3]
@Coordi777 Hello! Thank you very much for your code and pre-trained model! The pre-trained model works very well. However,  when I used train.py to retrain on the SEN1-2 dataset, I encountered some problems:

1. In line 342 of the train.py file, it is mentioned:
```Python
noise_scheduler = DDPMSchedulerColor.from_pretrained(os.path.join(args.pretrained_model_path,"scheduler"))
```
**This seems to require loading a noise_scheduler from a predefined configuration file, but it doesn't appear to be provided in the code you provided.** I tried defining a custom noise scheduler and loading the SD1.5 noise scheduler, training for 80 epochs each time (GPU: A800, batch size: 48), but the resulting images were unsatisfactory. The images below show the results of these two training runs (the first and second rows are real SAR and optical images, respectively; the third row is generated using the manually defined noise scheduler; the fourth row is generated using the SD1.5 noise scheduler):
</br>
<p align="center">
<img width="518" height="260" alt="SAR_GT" src="https://github.com/user-attachments/assets/ebe8a3b8-8069-4c1b-9a81-7c4091cc3052" />
</br>
<em>SAR_GT</em>
</br>
<img width="518" height="260" alt="OPT_GT" src="https://github.com/user-attachments/assets/2c0c3b6b-bd7c-46a2-a65f-cb7fe158a6d0" />
</br>
<em>OPT_GT</em>
</br>
<img width="518" height="260" alt="Experiment 1" src="https://github.com/user-attachments/assets/f97d9bfe-286e-4706-aa19-33e2427f3a3d" />
</br>
<em>Manually defined noise scheduler</em>
</br>
<img width="518" height="260" alt="Experiment 2" src="https://github.com/user-attachments/assets/19a1a0fd-d0c4-41a5-9268-5351de31cde8" />
</br>
<em>Noise scheduler for sd1.5</em>
</br>
</p>
</br>
2. The hyperparameters for both experiments above use the config_example.yaml file (except for batch size), but they appear to be different from the hyperparameters used in your training. **I would be grateful if you could provide more details about hyperparameters.**</br>
3. I would be very grateful if you could provide a **pretrained weights of train.py**
